app/uk/gov/hmrc/individualsemploymentsapi/domain/v2/Employer.scala

```diff
@@ -32,32 +32,31 @@ object Employer {
       (JsPath \ "payeReference").readNullable[EmpRef] and
         (JsPath \ "name").readNullable[String] and
         (JsPath \ "address").readNullable[Address]
-    )(Employer.apply _),
+      ) (Employer.apply _),
     (
       (JsPath \ "payeReference").writeNullable[EmpRef] and
         (JsPath \ "name").writeNullable[String] and
         (JsPath \ "address").writeNullable[Address]
-    )(unlift(Employer.unapply))
+      ) (unlift(Employer.unapply))
   )
 
   def create(payeReference: Option[EmpRef], name: Option[String], address: Option[Address]): Option[Employer] =
     (payeReference, name, address) match {
       case (None, None, None) => None
-      case _                  => Some(new Employer(payeReference, name, address))
```

Comment:
NO big but we can usually leave in space where its aligning code. My preference is to have the space however; its not a requirement so this is fine.